### ISSUE TITLE
berryboot only reads not writes

### ISSUE THREAD

**Issue by AaronRays468** [maxnet/berryboot#299]
before 2 days it works perfect.

--Aaron

**Comment by maxnet** [comment]
What worked 2 days ago, and no longer does?

**Comment by AaronRays468** [comment]
yes

**Comment by AaronRays468** [comment]
it does not even download if it is downloaded if i restart it restores from before boot

sorry for my bad english

**Comment by maxnet** [comment]
Try a different SD card

**Comment by AaronRays468** [comment]
nope,i have downloded before 1 year.in first boot it works perfect on my raspberry pi 2

**Comment by maxnet** [comment]
SD card may have run out of write cycles since.

**Comment by AaronRays468** [comment]
okay.but i can not do anything

**Comment by AaronRays468** [comment]
nope.i have a hp sd card 1 year warnenty

**Comment by maxnet** [comment]
You may be able to backup your existing files to USB stick using the "backup" button in Berryboot, if you want to keep them.
Install Berryboot to new SD card (or reinstall your existing one), and restore the images from USB stick later by holding "add OS" button and selecting "from USB stick"
